scripts/webpack/webpack.prod.babel.js

```diff
@@ -49,7 +49,7 @@ if (fs.existsSync('./src/index.html')) {
   );
 }
 
-const publicPath = process.env.BUILD_PUBLIC_PATH;
+const publicPath = process.env.BUILD_PUBLIC_PATH || '/';
```

Comment:
Why do we need this change?

Reply:
Same as webpack.base.js. The env variables are becoming undefined, so they webpack build is failling. This change just proved a fallback value.